pr_testing/run-pr-hlt-p2-timing.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+echo "FAILED" > $WORKSPACE/testsResults/statusfile-hlt-p2-timing.log
+
+[ "${WORKSPACE}" != "" ]         || export WORKSPACE=$(pwd) && cd $WORKSPACE
+source $WORKSPACE/cms-bot/pr_testing/setup-pr-test-env.sh
+
+# Report test started
+mark_commit_status_all_prs 'hlt-p2-timing' 'pending' -u "${BUILD_URL}" -d "Running"
+
+# Do work
+timeout $TIMEOUT ${CMSSW_CVMFS_PATH}/src/HLTrigger/Configuration/python/HLT_75e33/test/runHLTTiming.sh 2>&1 | tee $WORKSPACE/hlt-p2-timing.log
+CHART_URL="/circles/web/piechart.php?data_name=pr-hlt-p2-timing&resource=time_thread&filter=${UPLOAD_UNIQ_ID}&dataset=${UPLOAD_UNIQ_ID}/Phase2Timing_resources"
```

Comment:
`pr-hlt-p2-timing` will not work. We need to upload the json files in the same directory as IBs

Reply:
Why mix IB and PR results?

Reply:
As far as I see, it's just a matter of adding a symlink

Reply:
so that one can see both IB and PR tests in the drop down

Reply:
OK.